src/auth/auth-api-request.ts

```diff
@@ -120,6 +120,37 @@ class AuthResourceUrlBuilder {
 }
 
 
+/** Tenant aware resource builder utility. */
+class TenantAwareAuthResourceUrlBuilder extends AuthResourceUrlBuilder {
+  /**
+   * The tenant aware resource URL builder constructor.
+   *
+   * @param {string} projectId The resource project ID.
+   * @param {string} version The endpoint API version.
+   * @param {string} tenantId The tenant ID.
+   * @constructor
+   */
+  constructor(protected projectId: string, protected version: string, protected tenantId: string) {
+    super(projectId, version);
+    // Inject tenant path in URL.
+    this.urlFormat = this.urlFormat.replace(
```

Comment:
It looks like `urlFormat` in the parent class is really just a constant. Can we just define a constant for this as well?